static/app/views/dashboards/controls.tsx

```diff
@@ -178,6 +182,35 @@ function Controls({
                 {t('Export Dashboard')}
               </Button>
             </Feature>
+            <Feature features="dashboards-favourite">
+              <Button
+                size="sm"
+                aria-label={'dashboards-favourite'}
+                icon={
+                  <IconStar
+                    color={isFavorited ? 'yellow300' : 'subText'}
```

Comment:
Subtext ends up being `gray300` but we should just use the colour value we need. That way, if subtext changes, this button doesn't change unexpectedly

```suggestion
                    color={isFavorited ? 'yellow300' : 'gray300'}
```